ui/.eslintrc.cjs

```diff
@@ -1,3 +1,22 @@
+/*
```

Comment:
These should each go to the commits that introduced the respective files.

That's because we have a policy that each commit in a PR should pass all checks also on its own, not just the PR as a whole. Not having these headers would temporarily break REUSE checks.